### PR TITLE
Linkage videos

### PR DESCRIPTION
**Pull request by pwalczysko**
Adding the cleanup of the walkthroughs and linikage as discussed

cc @jburel

### CODE REVIEW

File: Day_4/0_Python_API.ipynb [hunk 1]
```diff
@@ -9,6 +9,8 @@
     "\n",
     "The image (``id=6001247``) is referenced in the paper \"NesSys: a novel method for accurate nuclear segmentation in 3D\" published August 2019 in PLOS Biology: https://doi.org/10.1371/journal.pbio.3000388 and can be viewed online in the [Image Data Resource](https://idr.openmicroscopy.org/webclient/?show=image-6001247) (IDR).\n",
     "\n",
+    "A set of walkthroughs showing the [exploration of IDR via User Interface](Exploring_IDR_UI.ipynb) is available in both text and video versions. These walkthroughs show the abovementioned publication and images from it in a larger context inside IDR and how it is linked to external databases.\n",
```
Comment (jburel) [inline Day_4/0_Python_API.ipynb]:
```suggestion
    "A set of walkthroughs showing the [exploration of IDR via the User Interface](Exploring_IDR_UI.ipynb) is available in both text and video versions. These walkthroughs show the above mentioned publication and images from it in a larger context inside IDR and how it is linked to external resources.\n",
```

File: Day_4/Exploring_IDR_UI.ipynb [hunk 2]
```diff
@@ -7,75 +7,78 @@
    "source": [
     "# Explore IDR data using its User Interface\n",
     "\n",
-    "This notebook present three workflows demonstrating how to find relevant data and metadata in the Image Data Resource:\n",
+    "This notebook present three workflows demonstrating how to find relevant data and metadata in the Image Data Resource. Each walkthrough is also available as a video which you can find both on [Openmicroscopy YouTube channel](https://www.youtube.com/c/OpenMicroscopyEnvironment/videos) and [IDR homepage](http://idr.openmicroscopy.org/):\n",
```
Comment (jburel) [inline Day_4/Exploring_IDR_UI.ipynb]:
```suggestion
    "This notebook presents three workflows demonstrating how to find relevant data and metadata in the Image Data Resource. Each walkthrough is also available as a video which you can find both on the [Openmicroscopy YouTube channel](https://www.youtube.com/c/OpenMicroscopyEnvironment/videos) and [IDR homepage](http://idr.openmicroscopy.org/):\n",
```

File: Day_4/Exploring_IDR_UI.ipynb [hunk 3]
```diff
@@ -7,75 +7,78 @@
...
+    "  * Search for a *Name (IDR number)* (the default option)\n",
+    "  * Enter *62* into the second search box.\n",
     "  * Accept the suggestion and click on the thumbnail. \n",
-    "  * This gives you all the images contained in the idr0062 study.\n",
+    "  * This gives you all the images contained in the `idr0062` study.\n",
     " \n",
     "This workflow shows one of the main uses of IDR for data visibility, Quality Control for publishers and community. \n",
     "\n",
-    "In the idr0062 study, on the image http://idr.openmicroscopy.org/webclient/?show=image-6001239, we can see the attached `tiff` file with ROIs. These ROIs were extracted into OMERO ROIs (saved as masks), which can be seen overlaid onto the images.\n",
+    "In the `idr0062` study, on the image http://idr.openmicroscopy.org/webclient/?show=image-6001239, we can see the attached `tiff` file with ROIs. These ROIs were extracted into OMERO ROIs (saved as masks), which can be seen overlaid onto the images.\n",
```
Comment (jburel) [inline Day_4/Exploring_IDR_UI.ipynb]:
```suggestion
    "In the `idr0062` study, on the image https://idr.openmicroscopy.org/webclient/?show=image-6001239, we can see the attached `tiff` file with ROIs. These ROIs were extracted into OMERO ROIs (saved as masks), which can be seen overlaid onto the images.\n",
```

File: Day_4/Exploring_IDR_UI.ipynb [hunk 4]
```diff
@@ -7,75 +7,78 @@
...
     "\n",
-    "### From Gene to Phenotypes  (FYI only, not shown)\n",
+    "### From gene to phenotypes  (FYI only, not shown)\n",
     "* Start from the [IDR homepage](https://idr.openmicroscopy.org/).\n",
-    "* Select Attribute Gene. \n",
-    "* Search for KRAS. \n",
-    "* In the results, find the study idr0033A. \n",
-    "* This is a gene overexpression study by Rohban et al. \n",
+    "* Select Attribute `Gene`. \n",
+    "* Search for *KRAS*. \n",
```
Comment (jburel) [inline Day_4/Exploring_IDR_UI.ipynb]:
```suggestion
    "* Search for `KRAS`. \n",
```

**Comment by pwalczysko** [comment]
fixes pushed

**Pull request merged**